_episodes/02-licensing.md

```diff
@@ -22,8 +22,8 @@ keypoints:
 {: .callout}
 ![recommendation2]({{ page.root }}/fig/recommendation2.png)
 
-*Disclaimer: This topic concerns law and can have serious repercussions. If you have a specific question ask an expert. 
-This text is intended to give a short overview of licensing and is not legal counsel. For the purpose of education content is simplified. Do not soley base your decisions on this text.
+*Disclaimer: This topic concerns law and can have serious repercussions. If you have a specific question ask an expert.
```

Comment:
I'm a bit worried that this disclaimer may discourage people from licensing their code out of fear, even in situations where the risk is really very low (for example, if their university has clear guidelines to follow, or if it's a personal project). Arguably, using someone else's code is riskier than licensing your own, and we do that all the time. So maybe we can rephrase or remove this to be less scary?